src/SDKs/KeyVault/dataPlane/Microsoft.Azure.KeyVault.Cryptography/Algorithms/Rs256.cs

```diff
@@ -20,6 +20,7 @@ public class Rs256 : RsaSignature
         internal const string OID_OIWSEC_SHA384 = "2.16.840.1.101.3.4.2.2";
         internal const string OID_OIWSEC_SHA512 = "2.16.840.1.101.3.4.2.3";
 
+        internal const int SHA256_LENGTH = 32;
```

Comment:
Nit: I would include 'DIGEST' in the name of the constant.  SHA256_DIGEST_LENGTH for instance.